src/VisualStudio/IntegrationTests/App.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we name this `app.config` (all lower case) for consistency?  Makes it easier to develop on Mac / Linux to be consistent here.